src/argilla/server/routes.py

```diff
@@ -76,3 +76,8 @@
 api_router.include_router(users_v1.router, prefix="/v1")
 api_router.include_router(vectors_settings_v1.router, prefix="/v1")
 api_router.include_router(workspaces_v1.router, prefix="/v1")
+
+
+@api_router.route("/{full_path:path}")
+def endpoint_not_found_controller(request: Request):
+    raise HTTPException(status_code=404, detail=f"Endpoint {request.url.path!r} not found")
```

Comment:
Maybe it's interesting to don't add this to swagger API schema.

Reply:
Done. Using `include_in_schema=False`